ios/Podfile

```diff
@@ -37,7 +37,7 @@ target 'Backpack Native' do
   pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec', :modular_headers => false
 
   # RN Bridging Pods
-  pod 'Backpack', '~> 41.0'
+  pod 'Backpack', git: 'https://github.com/Skyscanner/backpack-ios.git', branch: 'BPK-4426-add-calendar-accessibility-strings'
```

Comment:
Is this so that we can test it works and will be removed once the iOS part is done?

Reply:
Yeah, I'll update this when we merge and release Skyscanner/backpack-ios#919